ffcv/transforms/__init__.py

```diff
@@ -9,11 +9,13 @@
 from .translate import RandomTranslate
 from .mixup import ImageMixup, LabelMixup, MixupToOneHot
 from .module import ModuleWrapper
+from .color_jitter import RandomBrightness, RandomContrast, RandomSaturation
```

Comment:
Is RandomBrightness really `color_jitter` maybe it should use a different file name ?

Reply:
[Torchvision ColorJitter operation](https://github.com/pytorch/vision/blob/main/torchvision/transforms/transforms.py#L1140-L1253) includes changing brightness, contrast, saturation and hue of an image.

I think most of the people who use ffcv may familiar with torchvision so I just included on color_jitter.py to avoid confusions.

Any suggestions are welcome from using different file name!